File: send.exs
{:ok, connection} = AMQP.Connection.open
{:ok, channel} = AMQP.Channel.open(connection)

AMQP.Queue.declare(channel, "hello")

AMQP.Basic.publish(channel, "", "hello", "Hello from Elixir")
IO.puts " [x] Sent a hello message"

AMQP.Connection.close(connection)

File: ruby/send.rb
#!/usr/bin/env ruby
require 'bunny'

# create connection
conn = Bunny.new
conn.start

# create channel
ch = conn.create_channel

# define queue
q = ch.queue('hello')

# send a message
ch.default_exchange.publish 'Hello from Ruby', routing_key: q.name
puts ' [x] Sent a hello message'

# close a connection
conn.close
